custom_components/places/coordinator.py

```
@@ -248,6 +254,61 @@ def publish_update(self) -> None:
             return
         self.async_set_updated_data(self.snapshot())
 
+    async def async_update_setting(self, key: str, value: str | bool) -> None:
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)